app/api/routers/breeze_buddy/telephony/callbacks/handlers.py

```
@@ -197,59 +299,207 @@ async def handle_callback_status(request: Request, provider: str) -> Response:
-    <Stream {noise_cancellation_attr} bidirectional="true" keepCallAlive="true" contentType="audio/x-mulaw;rate=8000">
-        {ws_url}
-    </Stream>
+    <Speak>An error occurred. Please try again later.</Speak>
+    <Hangup/>
+</Response>"""
+        return HTMLResponse(content=xml_content, media_type="application/xml")
+
+    # Start recording for all calls
+    try:
+        start_call_recording(call_uuid)
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)